components/nimbus/Cargo.toml

```diff
@@ -35,6 +35,7 @@ once_cell = "1"
 uniffi = { version = "^0.17", optional = true }
 chrono = { version = "0.4", features = ["serde"]}
 unicode-segmentation = "1.8.0"
+glean = "42.3.2"
```

Comment:
The Swift code and the Rust code should both be using the same Glean xcframework on iOS. I think that gets resolved when the app compiles? (looking into it to be sure)

Reply:
Looks like it may require Glean to be in the iOS megazord to be able to actually record things. That's a blocker to recording metrics, but doesn't break anything with building or the integration here.

Reply:
I'm adding that as a blocker on migration of the existing events to Rust and adding the health metrics.